src/lib.rs

```diff
@@ -27,8 +41,23 @@ pub enum NEARLedgerError {
     DeviceNotFound,
     /// Error occurred while exchanging with Ledger device
     APDUExchangeError(String),
+    /// Blind signature not supported
+    BlindSignatureNotSupported,
+    /// Blind signature disabled in ledger's app settings
+    BlindSignatureDisabled,
     /// Error with transport
     LedgerHIDError(LedgerHIDError),
+    /// Transaction is too large to be signed
+    BufferOverflow(OnlyBlindSigning),
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct OnlyBlindSigning(pub CryptoHash);
```

Comment:
I would use struct with a named field here to explicitly say what that CryptoHash is going to be.

Reply:
addressed in [commit](https://github.com/khorolets/near-ledger-rs/pull/4/commits/a179ffa6791569cb599f630bab163bf1129f9c4c)

Reply:
addressed in [commit](https://github.com/khorolets/near-ledger-rs/pull/4/commits/26d0f39f09c5d59f7d9a8c029041a795022d4c67)